include/rocksdb/options.h

```diff
@@ -62,6 +62,8 @@ struct Options;
 struct DbPath;
 
 using FileTypeSet = SmallEnumSet<FileType, FileType::kBlobFile>;
+using CompactionStyleSet =
+    SmallEnumSet<CompactionStyle, CompactionStyle::kCompactionStyleUniversal>;
```

Comment:
`MAX_ENUMERATOR` should be the highest defined enumerator, even if it doesn't always make sense for various values to be in the set. (If a type definition appears to be generally useful, we shouldn't be special-case clever.)